src/snyk/base/modules/snykLib.ts

```diff
@@ -95,6 +95,13 @@ export default class SnykLib extends ReportModule implements ISnykLib {
       return;
     }
 
+    if (
+      !configuration.getFeaturesConfiguration()?.codeSecurityEnabled &&
+      !configuration.getFeaturesConfiguration()?.codeQualityEnabled
+    ) {
+      return;
```

Comment:
@michelkaporin , it's early exit to avoid Code scanning if Code functionality is not enabled, right?

Reply:
@pavel-snyk yes! wanted to leave a PR comment for that but forgot. It used not to respect the flag. I'd make a test for it but the class is not unit-testable, thus little out of scope for this PR.